resources/services/sns/topics.go

```diff
@@ -138,6 +145,17 @@ func resolveTopicAttributes(ctx context.Context, meta schema.ClientMeta, resourc
 	if err != nil {
 		return diag.WrapError(err)
 	}
+
+	tagParams := sns.ListTagsForResourceInput{
+		ResourceArn: topic.TopicArn,
+	}
+	tags, err := svc.ListTagsForResource(ctx, &tagParams, func(o *sns.Options) {
+		o.Region = c.Region
+	})
+	if err != nil {
+		return diag.WrapError(err)
+	}
+
```

Comment:
nit: would maybe move this code to a [ColumnResolver](https://docs.cloudquery.io/docs/developers/sdk/table/column-resolvers)

Reply:
Thanks for the feedback. I've made some changes to move this to a ColumnResolver